.github/workflows/main.yml

```diff
@@ -31,18 +35,20 @@ jobs:
       run: |
         poetry install
 
-    - name: Test with pytest
-      run: |
-        poetry run pytest --cov
-
     - name: Lint with mypy
       run: |
         poetry run mypy mousebender
-
+  
```

Comment:
```suggestion

```

Reply:
Not sure what you meant here - I like to separate my tasks with whitespace but I can remove it all if you prefer. (I actually like to put a comment on each task as well, but it hardly seems necessary with the way GH workflows work!).

Reply:
You inserted extraneous space. I didn't delete the newline, I just trimmed the line.

Reply:
Understood - I must've had a code formatter for the YAML in the latest revision work I was doing because there wasn't even spaces in the editor when I looked.